src/overlays/actors/ovl_En_Ik/z_en_ik.c

```diff
@@ -1058,48 +1107,56 @@ void func_80A770C0(EnIk* this, GlobalContext* globalCtx, s32 actionIdx) {
     }
 }
 
-f32 EnIk_curFrame(Actor* thisx) {
+f32 EnIk_CurFrame(Actor* thisx) {
     EnIk* this = (EnIk*)thisx;
 
     return this->skelAnime.curFrame;
 }
 
-void func_80A77148(EnIk* this) {
-    this->action = 0;
-    this->drawMode = 0;
+// unused
+void EnIk_CsAdvanceTo01(EnIk* this) {
+    this->action = IK_ACTION_INIT;
+    this->drawMode = IK_DRAW_NOTHING;
     this->actor.shape.shadowAlpha = 0;
 }
 
-void func_80A77158(EnIk* this, GlobalContext* globalCtx) {
-    Animation_Change(&this->skelAnime, &object_ik_Anim_00C114, 1.0f, 0.0f,
-                     Animation_GetLastFrame(&object_ik_Anim_00C114), ANIMMODE_ONCE, 0.0f);
-    func_80A770C0(this, globalCtx, 4);
-    this->action = 1;
+// Cutscene: Nabooru sitting and Kotake and Koume looking at her
+void EnIk_CsAdvanceTo02(EnIk* this, GlobalContext* globalCtx) {
+    Animation_Change(&this->skelAnime, &gIronKnuckleNabooruSummonAxeAnim, 1.0f, 0.0f,
+                     Animation_GetLastFrame(&gIronKnuckleNabooruSummonAxeAnim), ANIMMODE_ONCE, 0.0f);
+    EnIk_MoveNpcToPos(this, globalCtx, 4);
+    this->action = IK_ACTION_1;
     this->drawMode = 1;
```

Comment:
```suggestion
    this->drawMode = IK_DRAW_1;
```
and other `drawMode` usage